omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-SummaryPage.xml

```diff
@@ -0,0 +1,1732 @@
+                    } else {
+                        $firstRow.show();
+                        $firstRow.find('.remove').hide();
+                    }
+                });
+            </ifMode>
+            });
+
+        }
+    </script>
+
+    <script type="text/javascript">
+        function val() {
+            document.getElementById("fgh").disabled = "true";
+        }
+    </script>
+
+    <style type="text/css">
+        .error {
+            color: red;
+        }
+
+        form#htmlform div.ai-page-frame table tbody td {
+            /*background: #B6D6E6;*/
+            color: #404040;
+            padding: 0px 5px;
+        }
+
+        form select, .form select {
+            min-width: 0px;
+        }
+
+        form input, form select, form textarea, form ul.select, .form input, .form select, .form textarea, .form
+        ul.select {
+            background-color: #FBFBFB;
+            border: 1px solid #404040;
+            border-radius: 5px;
+            margin-bottom: 10px;
+            padding: 10px;
+            min-width: 50%;
+        }
+
+        .row {
+            width: 100%;
+            float: left;
+            display: inline-flex;
+            clear: both;
+        }
+
+        .row .row-col2 {
+            float: left;
+            width: 50%;
+        }
+
+        table th, table td {
+            /*border: 1px solid #BDBDBD;*/
+            text-align: left;
+            vertical-align: top;
+        }
+
+        .form-row {
+            display: -ms-flexbox;
+            display: flex;
+            -ms-flex-wrap: wrap;
+            flex-wrap: wrap;
+            margin-right: -5px;
+            margin-left: -5px;
+        }
+
+        .form-row > .col,
+        .form-row > [class*="col-"] {
+            padding-right: 5px;
+            padding-left: 5px;
+        }
+
+        select, .hasDatepicker,input[type=text]{
+            width:200px;
+        }
+    </style>
+
+    <!-- Beging of HTML Form -->
+    <div class="ai-page-frame">
+        <div class="headers" style="text-align: center;">
+            <h3>HIV Care/ART CARD - Summary Page</h3>
+        </div>
+        <div id="tabs">
+
+            <ul>
+                <li>
+                    <a href="#care-entry-section">
+                        Care Entry &amp; Family
+                    </a>
+                </li>
+                <li>
+                    <a href="#art-care-cohort-section">
+                        ART Care Cohort
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#prior-art-section">
+                        Treatment Interruptions
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#drug-allergies-and-art-care">
+                        Drug Allergies &amp; ART Care
+                    </a>
+                </li>
+            </ul>
+
+            <div id="care-entry-section" style="background: #F7F7F7;">
```

Comment:
@dbaluku Remove inline CSS background color - add a class art-care-card (with the background color)